Userland/Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -419,7 +418,7 @@ void HTMLElement::click()
     // 3. Set this element's click in progress flag.
     m_click_in_progress = true;
 
-    // FIXME: 4. Fire a synthetic pointer event named click at this element, with the not trusted flag set.
```

Comment:
Drive-by FIXME removal? Seems though as it should have been removed at some point

Reply:
Yeah, I almost removed it in a different commit, but I _think_ it must have been left open since it says specially "pointer event", which this now fixes?

Reply:
Ah, yes it does. I just read the method name and thought somebody had forgot to remove it previously.